packages/docs/src/components/global/DocRoadMapItem.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+	<div>
+		<VRow
+			no-gutters
+			class="mb-3"
+		>
+			<div class="font-weight-bold">
+				{{ item.title }}
+			</div>
+
+			<VRow
+				no-gutters
+				align="center"
+			>
+				<span :class="`ml-0 ml-sm-8 mt-2 mt-sm-0 px-3 rounded-pill text-caption white--text ${item.labelColor}`">{{ item.label }}</span>
+			</VRow>
+		</VRow>
+
+		<div>
+			{{ item.description }}
+		</div>
+
+		<div
+			v-if="item.issue && issueNumber"
+			class="mt-1"
+		>
+			<a
+				:href="item.issue"
+				target="_blank"
```

Comment:
```suggestion
				target="_blank"
				rel="noopener noreferrer"
```
Quand on met `target="_blank"` sur un lien c'est une bonne pratique de sécurité d'ajouter l'attribut `rel="noopener noreferrer"` (un article qui en parle : https://medium.com/sedeo/how-to-fix-target-blank-a-security-and-performance-issue-in-web-pages-2118eba1ce2f)

Reply:
Hm je vais lire ça, merci !